src/test/java/com/hyperwallet/clientsdk/util/HyperwalletApiClientTest.java

```diff
@@ -971,4 +1001,37 @@ public void testPost_500Response_WithRelatedResources() throws JSONException {
             assertThat(e.getResponse(), is(notNullValue()));
         }
     }
+
```

Comment:
Missing test where the test is passing with content-type response = `application/jose+json` ?

Reply:
Added test